example/src/androidTest/java/io/honeycomb/opentelemetry/android/example/HoneycombSmokeTest.kt

```diff
@@ -48,6 +48,11 @@ class HoneycombSmokeTest {
         rule.onNodeWithText("Send Metric").performClick()
     }
 
+    @Test
+    fun network_instrumentation_works() {
+        rule.onNodeWithText("Make a Network Request").performClick()
```

Comment:
unfortunately, this is gonna be flaky, since the button just _starts_ the request. I think we're gonna have to make the UI update when the request starts and ends, so that we can wait for the UI to update here before considering the test finished.

I would probably put a `Text` element in the UI and a `remember` state variable for the text to display. Then in this test, I would try to use `waitUntil` until the text matches.

https://developer.android.com/develop/ui/compose/testing/synchronization#manual-sync-waituntil